app/players/elves/druid.py

```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def get_rating(self) -> int:
+        return len(self._favourite_spell)
+
+    def player_info(self) -> str:
+        return (f"Druid {self.nickname}."
+                f" {self.nickname}"

```

Comment:
Code Style: It seems like you are duplicating the nickname in the player_info() method. You might want to remove the duplicate.